.github/workflows/ci.yml

```diff
@@ -91,26 +92,28 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+    - shell: bash
+      run: |
+        git fetch --prune --unshallow
```

Comment:
Why do you need to do this?

Reply:
I think that is for min-ver @thefringeninja ?